diff -Nru sqlite.orig/Makefile.in sqlite/Makefile.in
--- sqlite.orig/Makefile.in	Wed Jan  9 04:17:09 2002
+++ sqlite/Makefile.in	Tue Jan 29 13:27:37 2002
@@ -12,6 +12,8 @@
 # at the beginning of configure.in for additional information.
 #
 
+DESTDIR =
+
 # The toplevel directory of the source tree.  This is the directory
 # that contains this "Makefile.in" and the "configure.in" script.
 #
@@ -301,9 +303,9 @@
 	mv $(DOC) doc
 
 install:	sqlite libsqlite.la sqlite.h
-	$(LIBTOOL) $(INSTALL) libsqlite.la $(prefix)/lib
-	$(LIBTOOL) $(INSTALL) sqlite $(prefix)/bin
-	$(INSTALL) -m 0644 sqlite.h $(prefix)/include
+	$(LIBTOOL) $(INSTALL) libsqlite.la $(DESTDIR)$(prefix)/lib
+	$(LIBTOOL) $(INSTALL) sqlite $(DESTDIR)$(prefix)/bin
+	$(INSTALL) -m 0644 sqlite.h $(DESTDIR)$(prefix)/include
 
 clean:	
 	rm -f *.lo *.la *.o sqlite libsqlite.la sqlite.h
