diff -burN sqlite.orig/Makefile.in sqlite/Makefile.in
--- sqlite.orig/Makefile.in	2004-01-06 13:22:10.000000000 +0100
+++ sqlite/Makefile.in	2004-01-06 13:24:21.000000000 +0100
@@ -12,6 +12,8 @@
 # at the beginning of configure.in for additional information.
 #
 
+DESTDIR =
+
 # The toplevel directory of the source tree.  This is the directory
 # that contains this "Makefile.in" and the "configure.in" script.
 #
@@ -34,6 +34,11 @@
 #
 prefix = @prefix@
 exec_prefix = @exec_prefix@
+libdir = @libdir@
+bindir = @bindir@
+includedir = @includedir@
+mandir = @mandir@
+
 INSTALL = @INSTALL@
 LIBTOOL = ./libtool
 RELEASE = @ALLOWRELEASE@
@@ -190,12 +190,12 @@
           | awk '{print $$5,$$6}' >last_change
 
 libsqlite.la:	$(LIBOBJ)
-	$(LTLINK) -o libsqlite.la $(LIBOBJ) ${RELEASE} -rpath @exec_prefix@/lib \
+	$(LTLINK) -o libsqlite.la $(LIBOBJ) ${RELEASE} -rpath $(libdir) \
 		-version-info "8:6:8"
 
 libtclsqlite.la:	tclsqlite.lo libsqlite.la
 	$(LTLINK) -o libtclsqlite.la tclsqlite.lo \
-		libsqlite.la $(LIBTCL) -rpath @exec_prefix@/lib/sqlite \
+		libsqlite.la $(LIBTCL) -rpath $(libdir)/sqlite \
 		-version-info "8:6:8"
 
 sqlite@TARGET_EXEEXT@:	$(TOP)/src/shell.c libsqlite.la sqlite.h
@@ -460,14 +465,16 @@
 	mv $(DOC) doc
 
 install:	sqlite libsqlite.la sqlite.h
-	$(INSTALL) -d $(DESTDIR)/$(exec_prefix)/lib
-	$(LTINSTALL) libsqlite.la $(DESTDIR)/$(exec_prefix)/lib
-	$(INSTALL) -d $(DESTDIR)/$(exec_prefix)/bin
-	$(LTINSTALL) sqlite $(DESTDIR)/$(exec_prefix)/bin
-	$(INSTALL) -d $(DESTDIR)/$(prefix)/include
-	$(INSTALL) -m 0644 sqlite.h $(DESTDIR)/$(prefix)/include
-	$(INSTALL) -d $(DESTDIR)/$(exec_prefix)/lib/pkgconfig; 
-	$(INSTALL) -m 0644 sqlite.pc $(DESTDIR)/$(exec_prefix)/lib/pkgconfig; 
+	$(INSTALL) -d $(DESTDIR)$(libdir)
+	$(LTINSTALL) libsqlite.la $(DESTDIR)$(libdir)
+	$(INSTALL) -d $(DESTDIR)$(bindir)
+	$(LTINSTALL) sqlite $(DESTDIR)$(bindir)
+	$(INSTALL) -d $(DESTDIR)$(includedir)
+	$(INSTALL) -m 0644 sqlite.h $(DESTDIR)$(includedir)
+	$(INSTALL) -d $(DESTDIR)$(libdir)/pkgconfig; 
+	$(INSTALL) -m 0644 sqlite.pc $(DESTDIR)$(libdir)/pkgconfig; 
+	$(INSTALL) -d $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -m 0644 sqlite.1 $(DESTDIR)$(mandir)/man1
 
 clean:	
 	rm -f *.lo *.la *.o sqlite@TARGET_EXEEXT@ libsqlite.la sqlite.h opcodes.*
